framework/helpers/BaseConsole.php

```diff
@@ -1022,6 +992,60 @@ public static function updateProgress($done, $total, $prefix = null)
         flush();
     }
 
+    /**
+     * Return screen width
```

Comment:
It is not screen width. Screen width was returned by `static::getScreenSize()` inside.